packages/shared-metrics/src/util/nativeModuleRetry.js

```diff
@@ -171,51 +170,68 @@ function copyPrecompiled(opts, loaderEmitter, callback) {
 
     logger.info(`Found a precompiled version for ${opts.nativeModuleName} ${label}, unpacking.`);
 
-    tar
-      .x({
-        cwd: os.tmpdir(),
-        file: precompiledTarGzPath
-      })
-      .then(() => {
-        // See below for the reason why we append 'precompiled' to the path.
-        const targetDir = path.join(opts.nativeModulePath, 'precompiled');
+    let tar;
+
+    // tar 6.x supports Node >= 10
+    // Ensure we don't bring down customers application
+    try {
+      tar = require('tar');
+    } catch (err) {
+      logger.warn('Requiring tar@6.x has failed.', err);
```

Comment:
*suggestion:* We might want to mention what the user-observable consequence of the failure to require tar@6 is.

Maybe something like 

```
logger.warn(`Using the precompiled build for ${opts.nativeModuleName} ${label} is not supported on Node.js ${process.version}.`);
```